src/rest-server/src/models/job.js

```diff
@@ -394,6 +467,16 @@ class Job {
         appLaunchedTime: frameworkStatus.applicationLaunchedTimestamp,
         appCompletedTime: frameworkStatus.applicationCompletedTimestamp,
         appExitCode: frameworkStatus.applicationExitCode,
+        appExitSpec: this.generateExitInfo(frameworkStatus.applicationExitCode),
+        appExitMessages: {
+          container: this.extractContainerStderr(frameworkStatus.applicationExitDiagnostics),
+          runtime: this.extractRuntimeOutput(frameworkStatus.applicationExitDiagnostics),
+          launcher: this.extractLauncherOutput(frameworkStatus.applicationExitDiagnostics),
+        },
+        appExitTriggerMessage: frameworkStatus.applicationExitTriggerMessage,
+        appExitTriggerTaskRoleName: frameworkStatus.applicationExitTriggerTaskRoleName,
+        appExitTriggerTaskRoleIndex: frameworkStatus.applicationExitTriggerTaskRoleIndex,
+        // deprecated
         appExitDiagnostics: frameworkStatus.applicationExitDiagnostics,
         appExitType: frameworkStatus.applicationExitType,
```

Comment:
Have you changed other components, to not use the deprecated anymore?

Reply:
yes

Reply:
yes